src/pages/library/DataBrowserDetails.js

```diff
@@ -31,22 +36,130 @@ const getSnapshot = id => new Promise(resolve => setTimeout(() => {
+          div(['Eric Miron']),
+          div(['University of Chicago Medical Center']),
+          h(Link, { href: `mailto:fakeemail@fake.org` }, ['fakeemail@fake.org'])
+        ]), div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Data curator']),
+          div(['Will add later, after data structure is added'])
+        ]), div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Contributors']),
+          ..._.map(contributor => div(contributor), snapshot.contributors)
+        ]), div({ style: styles.attributesColumn }, [
+          h3({ style: styles.headers }, ['Publications']),
+          div(['Will add later, after data structure is added'])
+        ])
+      ])
+    ])
   ])
 }
 
-const Sidebar = () => {
-  return div({ style: { ...styles.content, width: 400, flexShrink: 0 } }, [
-    'Sidebar Content Area'
+const Sidebar = snapshot => {
+  return div({ style: { ...styles.content, width: 300, flexShrink: 0, display: 'flex', flexDirection: 'column', alignItems: 'center' } }, [
+    h2({ className: 'sr-only' }, ['Snapshot Data Details']),
+    div({ style: { backgroundColor: 'white', padding: 20, paddingTop: 0, width: '100%', border: '2px solid #D6D7D7', borderRadius: 5 } }, [
+      div([
+        h3(['Access type']),
+        div([
+          snapshot.locked ?
+            h(ButtonSecondary, {
+              style: { fontSize: 16, textTransform: 'none', height: 'unset' },
+              onClick: () => console.log('clicked')
+            }, [
+              div({ style: { display: 'flex', alignItems: 'center', justifyContent: 'center' } }, [
+                icon('lock', { size: 18, style: { marginRight: 10, color: colors.accent() } }), 'Request Access'
+              ])
+            ]) :
+            div([icon('lock-o', { size: 18, style: { marginRight: 10, color: colors.primary() } }), 'Access Granted'])
+        ])
+      ]), div([
+        h3({ style: styles.headers }, ['Donor size']),
+        div([(snapshot.donor.size || 0).toLocaleString()])
+      ]), div([
+        h3({ style: styles.headers }, ['Sample size']),
+        div([(snapshot.sample.size || 0).toLocaleString()])
+      ]), div([
+        h3({ style: styles.headers }, ['Donor modality']),
+        div([snapshot.donor.modality])
+      ]), div([
+        h3({ style: styles.headers }, ['Data type']),
+        div([snapshot.dataType])
+      ]), div([
+        h3({ style: styles.headers }, ['File counts']),
+        table([
+          tbody(
+            [..._.map(filetype => {
+              return tr([
+                td({ style: { paddingRight: 30 } }, [filetype]),
+                td([snapshot.files.types[filetype].toLocaleString()])
+              ])
+            }, Object.keys(snapshot.files.types)),
+            tr({ style: { fontWeight: 'bold', borderTop: '2px solid rgba(0,0,0,.3)' } }, [
+              td(['Total']),
+              td([snapshot.files.count.toLocaleString()])
+            ])]
```

Comment:
Same comment on the stylistic changes
```suggestion
      ]),
      div([
        h3({ style: styles.headers }, ['Donor size']),
        div([(snapshot.donor.size || 0).toLocaleString()])
      ]),
      div([
        h3({ style: styles.headers }, ['Sample size']),
        div([(snapshot.sample.size || 0).toLocaleString()])
      ]),
      div([
        h3({ style: styles.headers }, ['Donor modality']),
        div([snapshot.donor.modality])
      ]),
      div([
        h3({ style: styles.headers }, ['Data type']),
        div([snapshot.dataType])
      ]),
      div([
        h3({ style: styles.headers }, ['File counts']),
        table([
          tbody([
            ..._.map(filetype => {
              return tr([
                td({ style: { paddingRight: 30 } }, [filetype]),
                td([snapshot.files.types[filetype].toLocaleString()])
              ])
            }, _.keys(snapshot.files.types)),
            tr({ style: { fontWeight: 'bold', borderTop: '2px solid rgba(0,0,0,.3)' } }, [
              td(['Total']),
              td([snapshot.files.count.toLocaleString()])
            ])
          ])
```